@ndlib/gatsby-theme-marble/src/components/Shared/SearchBox/SearchField/index.js

```diff
@@ -25,7 +25,7 @@ export const SearchField = ({ submitSearch, searchReducer, searchPath, dispatch,
         placeholder={fieldLabel}
         onChange={(e) => dispatch(updateInput(e.target.value))}
         value={decodeURIComponent(rawInput)}
-        aria-label={fieldLabel}
+        aria-label='Search tool,'
```

Comment:
I know it looks weird but I added the comma because the Voiceover would announce the tool with a run-on sentence and it sounded pretty bad. The comma makes it pause.
![Screen Shot 2021-08-11 at 2 05 43 PM](https://user-images.githubusercontent.com/56038074/129080700-2878bea9-0b33-43c7-a4aa-0ff02f9b769a.png)